src/main.rs

```diff
@@ -79,6 +84,27 @@ and https://docs.joinplu.me/installation/init for more info.
     .expect("Error setting Ctrl-c handler");
 
     let mail = mail::init();
+
+    HttpServer::new(|| {
+        ActixApp::new().service(
+            aweb::scope("/")
+                .service(api::service())
+                .service(web::service())
+                .service(
+                    // TODO: caching and co.
+                    actix_files::Files::new("/static", "./static"),
+                )
+                .default_service(
+                    // TODO: real error page
+                    aweb::route().to(|| HttpResponse::NotFound()),
+                ),
+        )
+    })
+    .bind("127.0.0.1:7878")?
+    .run()?;
+
     println!("Welcome to the async experiment of Plume.");
     println!("If you can only see this message, we're not done yet.");
```

Comment:
we could now turn this into a log message now 😅

Reply:
I don't added `log` yet! But yes, we should do it at some point.